halpybot/commands/caseutils.py

```diff
@@ -58,41 +97,539 @@ async def cmd_go(ctx: Context, args: List[str]):
+        new_item=new_plt,
+        enum=True,
+    )
+    if update:
+        return await ctx.reply(update)
+
+
+@Commands.command("planet")
+@Require.permission(Drilled)
+@Require.channel()
+@CaseUtils.gather_case(2)
+async def cmd_planet(ctx: Context, args: List[str], case: Case):
+    """
+    Change the planet of an active KF case
+
+    Usage: !planet [board ID] [new system]
+    Aliases: n/a
+    """
+    if case.case_type != CaseType.FISH:
+        return await ctx.reply("Planet can't be changed for non-Fisher case")
+    newplan = await sys_cleaner(" ".join(args[1:]))
+    update = await update_single_elem_case_prep(
+        ctx=ctx, case=case, action="Client Planet", new_key="planet", new_item=newplan
+    )
+    if update:
+        return await ctx.reply(update)
+
+
+@Commands.command("casecoords")
+@Require.permission(Drilled)
+@Require.channel()
+@CaseUtils.gather_case(3)
+async def cmd_coords(ctx: Context, args: List[str], case: Case):
+    """
+    Change the coords of an active KF case
+
+    Usage: !casecoords [board ID] [X Coordinate Float] [Y Coordinate Float]
+    Aliases: n/a
+    """
+    if case.case_type != CaseType.FISH:
+        return await ctx.reply("Coordinates can't be changed for non-Fisher case")
+    try:
+        xcoord = float(args[1].strip())
+        ycoord = float(args[2].strip())
+    except ValueError as val_err:
+        raise ValueError("KF Coordinates Improperly Formatted") from val_err
+    newcoords = KFCoords(xcoord, ycoord)
+    update = await update_single_elem_case_prep(
+        ctx=ctx,
+        case=case,
+        action="Client Coordinates",
+        new_key="pcoords",
+        new_item=newcoords,
+    )
+    if update:
+        return await ctx.reply(update)
+
+
+@Commands.command("o2time")
+@Require.permission(Drilled)
+@Require.channel()
+@CaseUtils.gather_case(2)
+async def cmd_oxtime(ctx: Context, args: List[str], case: Case):
+    """
+    Change the remaining oxygen timer for a case
+
+    Usage: !o2time [board ID] [O2 Time in NN:NN]
+    Aliases: n/a
+    """
+    if case.case_type not in (CaseType.BLACK, CaseType.BLUE):
+        return await ctx.reply("O2 Timer Irrelevant - Canopy Not Breached.")
+    pattern = r"^\d{2}:\d{2}$"
+    if not re.match(pattern, args[2].strip()):
+        return await ctx.reply("Invalid O2 Time Given. Does not match pattern ##:##.")
+    update = await update_single_elem_case_prep(
+        ctx=ctx,
+        case=case,
+        action="O2 Timer",
+        new_key="o2_timer",
+        new_item=args[1].strip(),
+    )
+    if update:
+        return await ctx.reply(update)
+
+
+@Commands.command("synth")
+@Require.permission(Drilled)
+@Require.channel()
+@CaseUtils.gather_case(2)
+async def cmd_synth(ctx: Context, args: List[str], case: Case):
+    """
+    Toggle if a CMDR has synths available for a given case.
+
+    Usage: !synth [board ID] [Yes/True/No/False]
+    Aliases: n/a
+    """
+    if args[1].casefold() in ("yes", "true"):
```

Comment:
You have a ready made [to_bool](https://www.attrs.org/en/stable/api.html#attrs.converters.to_bool) function you can use instead